content/blog/rueckblick-zum-tag-der-offenen-tuer-2018/contents+en.lr

```diff
@@ -0,0 +1,30 @@
+author: Hackerdan
+---
+beitrag:
+
+Some visitors came this time also from friendly Hackspaces around the Bodensee. Thus it became an international open day. Many interesting conversations about different topics took place during the whole day.
+
+
+Many thanks to the waffle bakers, goulash cookers, cake dispensers, photographers, filmmakers, lecturers, organizers, builders, cleaners, helpers, explainers, visitors - all of them.
+---
+body: On Saturday, 17 November 2018 the Toolbox Bodensee and its projects presented themselves. Also this year there was a densely packed lecture program of different topics. The quadrocopter flight show and soldering for children have almost become a tradition. New this year were the game developers with the Retrogames. Our new lounge with cool wall painting was used for the short lectures. Many colourful lights were in operation again.
```

Comment:
New this year were the game developers with the Retrogames -> The game developer group presenting their retro games was one of the new things this year.